packages/mongo-nav/src/user-menu/UserMenuTrigger.tsx

```diff
@@ -123,7 +125,7 @@ export default function UserMenuTrigger({
         {...buttonDataProp.prop}
         ref={buttonRef}
         className={cx(baseButtonStyles, {
-          [openBaseButtonStyle]: open,
+          [openBaseButtonStyle]: open || active,
```

Comment:
Hey Sean!

Everything else in this PR looks stellar, but I don't think we should make this specific change. It removes the change of state in the button when you open it on the associated pages, and I don't think the indicator is particularly useful on the button itself.

Would you mind taking this out? The PR looks ready to go otherwise!